shared/src/theme/index.ts

```diff
@@ -0,0 +1,17 @@
+import mustache from 'mustache'
+import fs from 'fs'
+import path from 'path'
+
+const generateThemedHTMLEmail = (emailThemeFields: {
+  body: string
+  agencyName?: string
+  agencyLogoURI?: string
+  unsubLink: string
+}): string => {
+  const htmlTemplate = fs.readFileSync(path.resolve(__dirname, './email-theme.mustache'), 'utf-8')
```

Comment:
Might be better to use the async version of `readFile`. Synchronous read blocks the entire Node.js event loop.

Reply:
Makes sense, updated